.pipelines/stages/jobs/steps/utils/download-ort.yml

```diff
@@ -47,7 +47,7 @@ steps:
       set -e -x
       ort_stable_version=$(echo $(ort_version) | cut -d- -f1)
       echo $ort_stable_version
-      cp libonnxruntime.so libonnxruntime.so.$ort_stable_version
+      ln -s libonnxruntime.so libonnxruntime.so.$ort_stable_version
```

Comment:
The order is wrong. 
```
ln -s <source> <target>
```
libonnxruntime.so should be a symlink.

Reply:
libonnxruntime.so here is source here, unless we want to rename the it to libonnxruntime.so.$ort_stable_version first.

Reply:
When you build a C/C++ program that uses onnxruntime, the program's binary will require loading libonnxruntime.so.***.***  (with the version number) instead of libonnxruntime.so.  However, the file you found from our nuget package is without the version number, because zip format doesn't support symlink. That's the reason why this extra step is needed.

Reply:
Read this： https://tldp.org/HOWTO/Program-Library-HOWTO/shared-libraries.html

Reply:
@snnn does this need updating?

Reply:
It is already updated?